src/radiation_system.hpp

```diff
@@ -32,22 +32,25 @@
 #include "simulation.hpp"
 #include "valarray.hpp"
 
-// physical constants in CGS units
-static constexpr double c_light_cgs_ = C::c_light;	    // cgs
-static constexpr double radiation_constant_cgs_ = C::a_rad; // cgs
-static constexpr double inf = std::numeric_limits<double>::max();
-static constexpr double Erad_zero = 1e-14;
+// Hyper parameters of the radiation solver
 
-// Hyper parameters of the IMEX time integrator for the radiation subsystem and matter-radiation exchange
+// Include O(beta tau) terms in the space-like components of the radiation
+static constexpr bool compute_G_last_two_terms = true;
 
-// IMEX PD-ARS scheme
+// Time integration scheme
+// IMEX PD-ARS
 static constexpr double IMEX_a22 = 1.0;
 static constexpr double IMEX_a32 = 0.5; // 0 < IMEX_a32 <= 0.5
-
 // SSP-RK2 + implicit radiation-matter exchange
 // static constexpr double IMEX_a22 = 0.0;
 // static constexpr double IMEX_a32 = 0.0;
 
+// physical constants in CGS units
+static constexpr double c_light_cgs_ = C::c_light;	    // cgs
+static constexpr double radiation_constant_cgs_ = C::a_rad; // cgs
+static constexpr double inf = std::numeric_limits<double>::max();
+static constexpr double Erad_zero = 1e-14;
```

Comment:
Is this the radiation energy floor? That should be a runtime parameter if possible.

Reply:
This is the radiation energy fraction floor for each individual group. I'll remove this and replace it with Energy floor in my next PR where I introduce multigroup diffusion test.

Reply:
Can you add a comment in the source explaining this?